lib/WP_Auth0_Import_Settings.php

```diff
@@ -22,12 +22,10 @@ public function import_settings() {
 		// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotValidated
 		if ( ! wp_verify_nonce( wp_unslash( $_POST['_wpnonce'] ?? '' ), self::IMPORT_NONCE_ACTION ) ) {
 			wp_nonce_ays( self::IMPORT_NONCE_ACTION );
-			exit;
```

Comment:
I know `wp_die` will make it quit. But according to the docs, [wp_nonce_ays](https://developer.wordpress.org/reference/functions/wp_nonce_ays/) will prompt for yes/no. If this was previously returning here, why is now this return no longer required?

Reply:
The documentation is a bit confusing on this, but wp_nonce_ays actually [calls wp_die internally too](https://core.trac.wordpress.org/browser/tags/5.6/src/wp-includes/functions.php#L3273). It just invokes some frontend UI for that prompt before it quits. So we can trim those exits safely here.